OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -37,19 +37,24 @@ using namespace std;
 //=============================================================================
 // CONSTRUCTOR(S) AND DESTRUCTOR
 //=============================================================================
-// default destructor, copy constructor, copy assignment
+// default destructor
 //_____________________________________________________________________________
+PrescribedForce::PrescribedForce()
+{
+    setNull();
+    constructInfrastructure();
+
+}
+
 /**
- * Default constructor.
+ * Convenience constructor.
  */
-PrescribedForce::PrescribedForce(Body* body)
+PrescribedForce::PrescribedForce(const PhysicalFrame& frame)
```

Comment:
can also delegate to the default constructor with ` : PrescribedForce()` and eliminate duplication of `setNull()` etc...